src/build/arg/mod.rs

```diff
@@ -4340,7 +4447,8 @@ impl<'help> fmt::Debug for Arg<'help> {
              aliases: {:?}, short_aliases: {:?}, possible_values: {:?}, value_names: {:?}, \
              number_of_values: {:?}, max_values: {:?}, min_values: {:?}, value_delimiter: {:?}, \
              default_value_ifs: {:?}, value_terminator: {:?}, display_order: {:?}, env: {:?}, \
-             unified_ord: {:?}, default_value: {:?}, validator: {}, validator_os: {} \
+             unified_ord: {:?}, default_value: {:?}, validator: {}, validator_os: {}, \
```

Comment:
Added a forgotten trailing comma here...

Reply:
I've been planning to rewrite it with [`DebugStruct`](https://doc.rust-lang.org/std/fmt/struct.DebugStruct.html) anyway, but never found time